cmd/ntrip-mqtt-gateway/gateway.go

```diff
@@ -120,30 +170,22 @@ func (gateway *Gateway) GetSourcetable(w http.ResponseWriter, r *http.Request) {
 // subscription for each request and streaming the data back to the client
 func (gateway *Gateway) GetMount(w http.ResponseWriter, r *http.Request) {
 	logger := r.Context().Value("logger").(*log.Entry)
+	subClient := r.Context().Value("mqttclient").(mqtt.Client)
 
-	// Check if mountpoint exists
-	mount, exists := gateway.Mounts[r.URL.Path[1:]]
+	// TODO: Consider moving this check to the AuthenticatorMiddleware, as it does
+	// not hide information because the sourcetable is a public list of available
+	// endpoints - saves making a connection to the broker. Could potentially use the
+	// same check in the case of PostMount as well.
+	mount, exists := gateway.Mounts[strings.ToLower(r.URL.Path[1:])]
```

Comment:
I'm still not sure why this needed to be ToLower, because the Mount.Name's are definitely all uppercase.

Reply:
i thought it might've been that MQTT topic names are case insensitive, but they don't seem to be. weird, not sure.

does it make sense for NTRIP requests to be case-insensitive - e.g. `AlIcE00AuS0`? i don't have a problem with that